gen/o365_mu

```diff
@@ -135,7 +139,12 @@ sub processMember {
 	my %memberAttributes = attributesToHash $memberData->getAttributes;
 
 	my $UCO = $memberAttributes{$A_UF_LOGIN};
+
+	my $disableForward = $memberAttributes{$A_UF_DISABLE_MAIL_FORWARD};
 	my $mailForward = $memberAttributes{$A_UF_O365_MAIL_FORWARD};
```

Comment:
If $disableForward is true $mailForward should be empty.

Reply:
Fixed.